Fix typo in the Evas XCB backend

This typo prevents the XCB backend from building.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/src/modules/engines/software_x11/evas_xcb_outbuf.c
===================================================================
--- a/src/modules/engines/software_x11/evas_xcb_outbuf.c
+++ b/src/modules/engines/software_x11/evas_xcb_outbuf.c
@@ -871,7 +871,7 @@
           }
         else
           {
-             int pixelb = evas_software_xcb_output_buffer_depth(obr->xob) / 8;
+             int pixelb = evas_software_xcb_output_buffer_depth(obr->xcbob) / 8;
              int run;
              int dstjump;
              
